Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:18.17.1
```

Comment:
1. Image built on Alpine in much smaller
2. There's no need in specifying minor version, let it update to the latest

Use `node:18-alpine`